ci_scripts/check_fauxpas.sh

```diff
@@ -19,4 +19,5 @@ fi
 
 echo "Linting with Faux Pas..."
 
-fauxpas check Stripe.xcodeproj/ --target "StripeiOSStatic" --configFile "./FauxPasConfig/main.fauxpas.json" --minErrorStatusSeverity Concern
+set -e fauxpas check Stripe.xcodeproj/ --target "StripeiOSStatic" --configFile "./FauxPasConfig/main.fauxpas.json" --minErrorStatusSeverity Concern
```

Comment:
Yeah I think they are doing so needlessly. :shrug:

On Fri, Aug 12, 2016 at 4:55 PM, Brian Dorfman <notifications@github.com>
wrote:

> In ci_scripts/check_fauxpas.sh
> <https://github.com/stripe/stripe-ios/pull/404#discussion_r74675595>:
>
> > @@ -19,4 +19,5 @@ fi
> >
> >  echo "Linting with Faux Pas..."
> >
> > -fauxpas check Stripe.xcodeproj/ --target "StripeiOSStatic" --configFile "./FauxPasConfig/main.fauxpas.json" --minErrorStatusSeverity Concern
> > +set -e fauxpas check Stripe.xcodeproj/ --target "StripeiOSStatic" --configFile "./FauxPasConfig/main.fauxpas.json" --minErrorStatusSeverity Concern
>
> Oh is that true? All of our other scripts redo the set every line so I
> assumed it only worked for the next command. I can change it.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/stripe/stripe-ios/pull/404/files/d52178f148b4e7addfcfd12cfe18428cbddbaeac#r74675595>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAtM5yhHXNXvVcD2GaMtFunlEAIijyXwks5qfQgKgaJpZM4Jjh8X>
> .
>